### ISSUE TITLE
Dust Off Github Repository

### ISSUE THREAD

**Issue by AnthonySMyers** [nnguyen682/CECS491ATeam4#35]
Begin committing to the repository and refresh our memories on what code we have already completed.

**Comment by AnthonySMyers** [comment]
We have reviewed the source code from last semester and have started editing it. As of right now we have created code that connects the Lambda Function to a DynamoDB Table. We have not committed the changed code to Github yet, but we will start working towards more frequent commits, pushes, and pulls. We are still figuring out code change management. Amazon does not allow easy links between accounts regarding tables and functions. Our plan is to test out a group account.

**Comment by AnthonySMyers** [comment]
I have uploaded two text files to our Github repository regarding DynamoDB connections and Psychotherapy methods.

[Steps for DynamoDB.txt](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a66619da7ba990013f053dc/125516c66e82c728ace21e0d46db809f3391c3af80fbff67bd53f199266205316546b3397f8fe62bab0021435a0e48fd100c0d12fea8812ba6e73c708d0a59edd4390b72d42103eee9dc4be05ce36ade0d97bcae8b4bcae93826e0888f4a381d06108dfc4f78.txt)

[Psychotherapy Notes.txt](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a66619da7ba990013f053dc/125516c66e82c728ace21e0d46d88783208a8cbd87ecad42b444b0ba265222606b48e13b628ae92dbc56395b4d5a1faa10090b16aafbd823a6b66d708a5b5dbfd1680b20c02f0dbefad341eb5ee779c11ac7a4b6984dc9e23c20e48c8c4f3b1800118afc5c27ed.txt)

**Comment by nnguyen682** [comment]
Because I have to format my hard drive couple months ago. I have to clone the repository again. And while I'm reviewing this. I noticed I commit the user manual html file with a non-sense description. So I recommitted it with a better description.

**Comment by nnguyen682** [comment]
I just pushed my files that contain the logic of the new feature: story. Right now if you enter a story then It will go to story intention slots and say this is story

**Comment by nnguyen682** [comment]
I also have been debugging the program and I found out that It stuck in the loop for the returning users that discussed their feelings. When the user say they have a bad day and logout and comeback again. The program stuck in the loop because we place the skill state after the alexa emit conversation.
In short I fixed it by moving setting the skill state back to null before the alexa emit conversation
